PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,23 @@
-Please make sure you've read the [pull request](http://docs.mattermost.com/developer/contribution-guide.html#preparing-a-pull-request) section of our [code contribution guidelines](http://docs.mattermost.com/developer/contribution-guide.html).
+<!--
+Please make sure you've read the pull request section (http://docs.mattermost.com/developer/contribution-guide.html#preparing-a-pull-request) of our code contribution guidelines (http://docs.mattermost.com/developer/contribution-guide.html).
 
-When filling in a section please remove the help text and the above text.
+REMEMBER TO:
+- Run `make check-style` to check for style errors (required for all pull requests)
+- Run `make test` to ensure unit tests passed
+- Run `make flow` to ensure type checking passed
+- Add or update unit tests (required for all new features)
+- All new/modified APIs include changes to the [JavaScript driver](https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.js)
+-->
 
 #### Summary
-[A brief description of what this pull request does.]
+<!--
+A brief description of what this pull request does.
+-->
 
 #### Ticket Link
-[Please link the GitHub issue or Jira ticket this PR addresses.]
-
-#### Checklist
-[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
-- [ ] Ran `make check-style` to check for style errors (required for all pull requests)
-- [ ] Ran `make test` to ensure unit tests passed
-- [ ] Ran `make flow` to ensure type checking passed
-- [ ] Added or updated unit tests (required for all new features)
-- [ ] All new/modified APIs include changes to the [JavaScript driver](https://github.com/mattermost/mattermost-redux/blob/master/src/client/client4.js)
+<!--
+Please link the GitHub issue or Jira ticket this PR addresses.
```

Comment:
I would like to just use the text from the server repo and encourage contributors to use `Fixes`: https://github.com/mattermost/mattermost-server/blob/master/.github/PULL_REQUEST_TEMPLATE.md#L15-L19